packages/frontend/src/ee/pages/AiAgents/AgentPage.tsx

```diff
@@ -158,7 +158,7 @@ const AgentPage = () => {
 
                     {projectUuid && threads && (
                         <Stack gap="xs">
-                            <Group justify="space-between">
+                            <Group justify="space-between" mih="24px">
```

Comment:
you can avoid this mih by using "visibility: hidden" on <Button> if there's a `threadUuid`